plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -319,18 +322,28 @@ protected String renameTableSql(String catalogName, String remoteSchemaName, Str
     }
 
     @Override
-    protected String renameColumnSql(JdbcTableHandle handle, JdbcColumnHandle jdbcColumn, String newRemoteColumnName)
+    public void renameColumn(ConnectorSession session, JdbcTableHandle handle, JdbcColumnHandle jdbcColumn, String newColumnName)
```

Comment:
The fact that overriding this method requires you do deal with identifierMapping and you didn't have to do it when overriding the previous one indicates the extension points are ill-defined.
Let me offer an improvement for this